README.md

```diff
@@ -270,6 +271,11 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
   
 ---
 
+## Linux
```

Comment:
Hey, Add this section above Machine learning section in alphabetical order

Reply:
sure

Reply:
done !